docs/desktop/layout/grid.md

```diff
@@ -37,6 +37,7 @@ Layout properties
 -   **column** *(Integer)*: The column of the cell the widget should occupy. Each cell can only contain one widget. This layout property is mandatory.
 -   **rowSpan** *(Integer)*: The number of rows, the widget should span, starting from the row specified in the `row` property. The cells in the spanned rows must be empty as well. (defaults to `1`)
 -   **colSpan** *(Integer)*: The number of columns, the widget should span, starting from the column specified in the `column` property. The cells in the spanned columns must be empty as well. (defaults to `1`)
+-   **allowGrowSpannedCellWidth** *(Boolean)*: Allow growing of spanning cells widths beyond the cumulated widths of the columns. The default behavior is that the width of the the spanning cell is determined by the cumulated width of the columns (plus spacing). Setting this property to true lets the cell width grow as needed to show the widget in the spanning cell, which also enlarges the width of the spanned columns. (defaults to `true`)
```

Comment:
add possessive apostrophe to "cells" so it becomes "Allow growing of spanning cells' widths..."

replace "cumulated" with "accumulated" throughout

double "the" in "is that the width of THE THE spanning cell"

Reply:
@derrell regarding your first comment: I meant here cells in plural. But you're the native speaker. should I change this then?

The other comments: will change that soon

Reply:
Yes. In English, the general rule for possessive of a single thing is the singular form followed by `'s`, so if it belongs to a single cell, it would be `cell's`. If you are talking about belong to all of the cells, then the apostrophe comes after the plural form, so in this case, `cells'`.

Reply:
Ah OK. Changed that.

Reply:
I mean changed it back again.